tasks/monitoring/all_in_one.yml

```diff
@@ -13,11 +13,18 @@
   with_items:
     - bbb_exporter_secrets.env
     - docker-compose.yaml
-    - prometheus.yaml
   notify: restart monitoring container
 
+- name: copy prometheus files into place
+  template:
+    src: "monitoring/prometheus.yaml.j2"
+    dest: "{{ bbb_monitoring_all_in_one_directory }}/prometheus.yaml"
+    mode: '0644'
+  notify: restart monitoring container
+  when: bbb_monitoring_all_in_one_enable | bool and not bbb_monitoring_external | bool
```

Comment:
In templates/monitoring/docker-compose.yaml.j2 the prometheus-container is still deployed.
This should be done with the same logic as here.